src/prepareData.js

```diff
@@ -1,12 +1,17 @@
 import Relay from 'react-relay';
 import toGraphQL from 'react-relay/lib/toGraphQL';
 
-export default function prepareData({ Container, queryConfig }, networkLayer) {
+export default function prepareData({ Container, queryConfig, preloadedRequests = [] }, networkLayer) {
   return new Promise((resolve, reject) => {
     const environment = new Relay.Environment();
-
     const data = [];
 
+    const storeData = environment.getStoreData();
+    preloadedRequests.forEach(request => {
```

Comment:
Nice! But more ES6-ish syntax here would line better to the rest of the codebase:
```js
for (const { query, response } of preloadedRequests) {
```